auslib/test/blobs/test_gmp.py

```diff
@@ -67,6 +80,9 @@ def testGetVendorsForPlatformOnlyInOne(self):
     def testGetResolvedPlatform(self):
         self.assertEquals("q", self.blob.getResolvedPlatform("c", "q2"))
 
+    def testGetResolvedPlatformDefault(self):
+        self.assertEquals("default", self.blob.getResolvedPlatform("d", "q2"))
+
```

Comment:
Lets add a test to ensure that we prefer a specified platform over default when both are present, eg 
  self.assertEquals("r", self.blob.getResolvedPlatform("d", "r"))